apps/andi/lib/andi_web/live/dataset_live_view/event_log_form.ex

```diff
@@ -0,0 +1,163 @@
+defmodule AndiWeb.EditLiveView.EventLogForm do
+  @moduledoc """
+  LiveComponent for viewing EventLog
+  """
+  use Phoenix.LiveView
+  import Phoenix.HTML.Form
+  import AndiWeb.Helpers.SortingHelpers
+
+  alias AndiWeb.ErrorHelpers
+  alias AndiWeb.DataDictionary.Tree
+  alias AndiWeb.InputSchemas.DataDictionaryFormSchema
+  alias Andi.InputSchemas.Datasets
+  alias Andi.InputSchemas.Datasets.DataDictionary
+  alias Andi.InputSchemas.DataDictionaryFields
+  alias Andi.InputSchemas.StructTools
+  alias Andi.InputSchemas.InputConverter
+  alias Ecto.Changeset
+
+  def mount(_, %{"dataset" => dataset, "order" => order}, socket) do
+    AndiWeb.Endpoint.subscribe("toggle-visibility")
+
+    event_log =
+      Andi.InputSchemas.EventLogs.get_all_for_dataset_id(dataset.id)
+      |> convert_to_string_keys
+      |> sort_list_by_field(:timestamp, "asc")
+
+    {:ok,
+     assign(socket,
+       visibility: "collapsed",
+       validation_status: "collapsed",
+       dataset: dataset,
+       dataset_id: dataset.id,
+       order: order,
+       event_log_order: %{"timestamp" => "asc"},
+       event_log: event_log
+     )}
+  end
+
+  def render(assigns) do
+    action =
+      case assigns.visibility do
+        "collapsed" -> "EDIT"
+        "expanded" -> "MINIMIZE"
+      end
+
+    ~L"""
+    <div id="event_log" class="form-component form-end">
+      <div class="component-header" phx-click="toggle-component-visibility" phx-value-component="event_log_form">
+        <div class="section-number">
+          <div class="component-number component-number--<%= @validation_status %>"><%= @order %></div>
+          <div class="component-number-status--<%= @validation_status %>"></div>
+        </div>
+        <div class="component-title full-width">
+          <h2 class="component-title-text component-title-text--<%= @visibility %> ">Event Log</h2>
+          <button aria-label="Event Log <%= action %>" type="button" class="btn btn--right btn--transparent component-title-button">
+            <div class="component-title-action">
+              <div class="component-title-action-text--<%= @visibility %>"><%= action %></div>
+              <div class="component-title-icon--<%= @visibility %>"></div>
+            </div>
+          </button>
+        </div>
+      </div>
+
+      <div class="component-edit-section--<%= @visibility %>">
+        <div class="event_log_table">
+          <table class="datasets-table" title="Event Log">
+            <thead>
+              <th class="datasets-table__th datasets-table__cell datasets-table__th--sortable datasets-table__th--<%= Map.get(@event_log_order, "timestamp", "unsorted") %>" phx-click="order-by" phx-value-field="timestamp">Timestamp</th>
```

Comment:
Are we defaulting values to "unsorted"? Would that display the word "unsorted" in the timestamp field if there isn't a value?